.github/workflows/debian_package_kanidm.yml

```diff
@@ -9,6 +9,10 @@ env:
   SCCACHE_GHA_ENABLED: "true"
   RUSTC_WRAPPER: "sccache"
 
+permissions:
```

Comment:
This needs to be applied to the single step which needs it, not the whole `.yml`